packages/@lightningjs/ui-components/src/components/Keyboard/Keyboard.stories.js

```diff
@@ -113,7 +113,7 @@ Keyboard.argTypes = {
     description: "Center the keys within it's set width of keyboard",
     control: 'boolean',
     table: {
-      defaultValue: { summary: false }
+      defaultValue: { summary: true }
```

Comment:
The default value for `Keyboard.centerKeys` is `undefined` in the `Keyboard` component. I think `false` would be the more accurate value here.